### PR TITLE
Move iOS post-commit queues to iOS 16

### PR DESCRIPTION
**Pull request by chgibb-apple**
#### fa76ac8c429bb6d8ef31919f6b33ab78d8e58a65
<pre>
Move iOS post-commit queues to iOS 16
<a href="https://bugs.webkit.org/show_bug.cgi?id=245155">https://bugs.webkit.org/show_bug.cgi?id=245155</a>
rdar://99766343

Reviewed by Ryan Haddad.

* Tools/CISupport/build-webkit-org/config.json:
* Tools/CISupport/build-webkit-org/factories_unittest.py:
(TestExpectedBuildSteps):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/IOS16.png: Added.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/IOS16Simulator.png: Added.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js:
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js:
(WebKitBuildbot):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css:
(table.queue-grid tr.platform.ios-simulator-16 img.logo):
(table.queue-grid tr.platform.ios-16 img.logo):

Canonical link: <a href="https://commits.webkit.org/254453@main">https://commits.webkit.org/254453@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/5564aeca6f4936684a77ee843e8b98f2cec936b7

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/89105 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/33666 "Built successfully") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/19936 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/98415 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/154737 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/93114 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/32174 "Built successfully") | [  ~~🛠 mac-debug~~](https://ews-build.webkit.org/#/builders/71/builds/27733 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/81465 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/92898 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/94752 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/68/builds/25542 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/61/builds/76040 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/81465 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/80403 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/19936 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/81465 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/29956 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/19936 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/20/builds/88161 "Passed tests") | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/67/builds/29684 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/19936 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/33128 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/61/builds/76040 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1311 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/31825 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/19936 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | |

### CODE REVIEW

File: Tools/CISupport/build-webkit-org/config.json [hunk 1]
```diff
@@ -738,17 +738,17 @@
...
                     },
-                    { "type": "PlatformSpecificScheduler", "platform": "ios-simulator-15", "branch": "main", "treeStableTimer": 45.0,
-                      "builderNames": ["Apple-iOS-15-Simulator-Release-Build", "Apple-iOS-15-Simulator-Debug-Build"]
+                    { "type": "PlatformSpecificScheduler", "platform": "ios-simulator-16", "branch": "main", "treeStableTimer": 45.0,
+                      "builderNames": ["Apple-iOS-16-Simulator-Release-Build", "Apple-iOS-16-Simulator-Debug-Build"]
                     },
-                    { "type": "PlatformSpecificScheduler", "platform": "tvos-15", "branch": "main", "treeStableTimer": 45.0,
-                      "builderNames": ["Apple-tvOS-15-Release-Build"]
+                    { "type": "PlatformSpecificScheduler", "platform": "tvos-16", "branch": "main", "treeStableTimer": 45.0,
+                      "builderNames": ["Apple-tvOS-16-Release-Build"]
```
Comment (ryanhaddad) [inline Tools/CISupport/build-webkit-org/config.json]: Find and replace went too far, don't think you intended to switch this yet.

File: Tools/CISupport/build-webkit-org/config.json [hunk 2]
```diff
@@ -738,17 +738,17 @@
...
                     },
-                    { "type": "PlatformSpecificScheduler", "platform": "tvos-simulator-15", "branch": "main", "treeStableTimer": 45.0,
-                      "builderNames": ["Apple-tvOS-Simulator-15-Release-Build"]
+                    { "type": "PlatformSpecificScheduler", "platform": "tvos-simulator-16", "branch": "main", "treeStableTimer": 45.0,
+                      "builderNames": ["Apple-tvOS-Simulator-16-Release-Build"]
```
Comment (ryanhaddad) [inline Tools/CISupport/build-webkit-org/config.json]: Find and replace went too far, don't think you intended to switch this yet.

File: Tools/CISupport/build-webkit-org/factories_unittest.py [hunk 3]
```diff
@@ -929,7 +929,7 @@ class TestExpectedBuildSteps(unittest.TestCase):
             'builtins-generator-tests',
             'trigger-crash-log-submission'
         ],
-        'Apple-tvOS-15-Release-Build': [
+        'Apple-tvOS-16-Release-Build': [
```
Comment (ryanhaddad) [inline Tools/CISupport/build-webkit-org/factories_unittest.py]: Find and replace went too far, don't think you intended to switch this yet.

File: Tools/CISupport/build-webkit-org/factories_unittest.py [hunk 4]
```diff
@@ -940,7 +940,7 @@ class TestExpectedBuildSteps(unittest.TestCase):
             'delete-stale-build-files',
             'compile-webkit'
         ],
-        'Apple-tvOS-Simulator-15-Release-Build': [
+        'Apple-tvOS-Simulator-16-Release-Build': [
```
Comment (ryanhaddad) [inline Tools/CISupport/build-webkit-org/factories_unittest.py]: Find and replace went too far, don't think you intended to switch this yet.

**Review by ryanhaddad — CHANGES_REQUESTED** [review]
I think fixing the tvOS errors will make unit tests pass, then this should be good to go (after squash + updating commit message format, of course)

File: Tools/CISupport/build-webkit-org/factories_unittest.py (hunk 4 above)
Comment (chgibb-apple) [reply Tools/CISupport/build-webkit-org/factories_unittest.py]: Indeed. The unit tests caught me on this as well.

**Review by ryanhaddad — APPROVED** [review]

**Pull request closed without merging**